client/platform/web-girder/views/ViewerLoader.vue

```diff
@@ -71,7 +78,24 @@ export default defineComponent({
       return parsed;
     });
     const { getters } = store;
-
+    const currentJob = computed(() => getters['Jobs/datasetCompleteJobs'](props.id));
+    watch(currentJob, async () => {
+      if (currentJob.value !== false && currentJob.value !== undefined) {
+        const result = await prompt({
+          title: 'Pipeline Finished',
+          text: [`Pipeline: ${currentJob.value.title}`,
+            'finished running sucesfully on the current dataset.  Click reload to load the annotations.  The current annotations will be replaced with the pipeline output.',
+          ],
+          confirm: true,
+          positiveButton: 'Reload',
+          negativeButton: 'Cancel',
```

Comment:
I think this was a leftover from before when a user could edit values while a pipeline was running. It made sense in very weird circumstances like you had a pipeline running for 3 hours and did a bunch of annotations and would rather keep those annotations and save them instead of overwriting them.  With the revision history and now locking to read only mode during pipeline runs I think I can remove the cancel.